### PR TITLE
FPGA Echo & Laser Transmission code

### PR DESCRIPTION
**Pull request by anju777**
To load onto FPGA, output_files -> LaserTransmission -> bat file -> type 3

### CODE REVIEW

**Comment by anju777** [comment]
To load Laser Transmission program onto FPGA, output_files -> LaserTransmission -> EchoLaserTransmission.bat (maybe Windows only)
Once command prompt opens, type '3' -> Enter

Board switch configurations:
KEY[0]: Reset
SW[0]: enable whole system
SW[1]: If on, echo mode on (if off, laser transmission mode)
SW[6]: Should turn ON (laser uses 2 modes of operation)
SW[7]: Force laser on (if SW[0] is on)
SW[8]: Constant transmission of 0xa
SW[9]: Turn ambient light off

tl;dr Just turn SW[0] and SW[6] on, everything else off. Reset at beginning.
